asr1k_neutron_l3/models/neutron/l3/vrf.py

```diff
@@ -34,18 +34,30 @@ def __init__(self, name, description=None, asn=None, rd=None, routable_interface
         self.asn = asn
         self.rd = utils.to_rd(self.asn, rd)
 
-        self.enable_bgp = False
-        self.map = "exp-{}".format(self.name)
+        # FIXME: hmmm might also need to be AF aware, but... when do we have a routable interface?
```

Comment:
```suggestion
        # NOTE: hmmm might also need to be AF aware, but... when do we have a routable interface?
```

Reply:
Uhm yeah, the comment in the code is just plain wrong. `self.map` is only used for v4 and v6 has its own map. I think that comment was added before this was the case (when I was still trying to figure out if I can do this with one route-map, which we could, but decided against it, due to other participants regarding this as a more complex solution).

I'll remove the offending comment completely.